service/resource/endpoint/create.go

```diff
@@ -57,7 +58,26 @@ func (r *Resource) ProcessCreateState(ctx context.Context, obj, createState inte
 	}
 
 	k8sEndpoint, err := r.k8sClient.CoreV1().Endpoints(endpointToCreate.ServiceNamespace).Get(endpointToCreate.ServiceName, metav1.GetOptions{})
-	if err != nil {
+	if errors.IsNotFound(err) {
```

Comment:
Subject to change in #12, this has to be done as the Endpoint might not exist when a new worker/master is created.